lib/improver/tests/ensemble_calibration/ensemble_calibration/test_EstimateCoefficientsForEnsembleCalibration.py

```diff
@@ -553,27 +643,22 @@ def setUp(self):
         self.wind_speed_truth_cube = (
             _create_truth(self.current_wind_speed_forecast_cube))
 
+        self.coeff_names = ["gamma", "delta", "alpha", "beta"]
+
     @ManageWarnings(
         ignored_messages=IGNORED_MESSAGES, warning_types=WARNING_TYPES)
     def test_basic(self):
         """Ensure that the optimised_coeffs are returned as a dictionary,
            and the coefficient names are returned as a list."""
-        current_forecast = self.current_temperature_forecast_cube
-
-        historic_forecasts = self.historic_temperature_forecast_cube
-
-        truth = self.temperature_truth_cube
-
         distribution = "gaussian"
-        desired_units = "degreesC"
+        current_cycle = "20171110T0000Z"
 
-        plugin = Plugin(distribution, desired_units)
+        plugin = Plugin(distribution, current_cycle)
         result = plugin.estimate_coefficients_for_ngr(
-            current_forecast, historic_forecasts, truth)
-        self.assertIsInstance(result, dict)
-        self.assertIsInstance(plugin.coeff_names, list)
-        for key in result.keys():
-            self.assertEqual(len(result[key]), len(plugin.coeff_names))
+            self.historic_temperature_forecast_cube,
+            self.temperature_truth_cube)
+        self.assertIsInstance(result, iris.cube.Cube)
+        self.assertArrayAlmostEqual(len(result.data), len(self.coeff_names))
```

Comment:
These tests don't match what is described in the doc string.

Reply:
I've updated the docstrings for these tests.